easybuild/easyconfigs/n/npstat/npstat-0.99_fix-hardcoding-Makefile.patch

```diff
@@ -0,0 +1,16 @@
+fix hardcoding of 'gcc'
+author: Kenneth Hoste (HPC-UGent)
+--- npstat/trunk/Makefile.orig	2016-02-15 01:16:51.000000000 +0100
++++ npstat/trunk/Makefile	2016-03-10 09:25:46.000000000 +0100
+@@ -1,8 +1,7 @@
+ all: npstat
+ 
+-
+ npstat: 
+-	gcc -o npstat NPStat-v0.99.c -lgsl -lgslcblas -lm
++	${CC} -o npstat NPStat-v0.99.c -lgsl -lgslcblas -lm
```

Comment:
good point about `$CFLAGS`, I'll add that too

in fact, since they're not specifying any optimisation flags, they were compiling with `-O0` (the default for GCC)

I bet that was a paper with very good results... >_<

The libraries (and headers) are handled through `$LIBRARY_PATH`.